test/wiki-report.py

```diff
@@ -117,4 +119,10 @@ def run(self):
         print("Usage: compose-report.py <report_path>")
         sys.exit(1)
 
+    if not os.getenv("COCKPIT_FEDORA_WIKI_TOKEN"):
+        print("Please set COCKPIT_FEDORA_WIKI_TOKEN environment variable with the file path to the wiki token.")
+        sys.exit(1)
+
+    shutil.copy(os.getenv("COCKPIT_FEDORA_WIKI_TOKEN"), "~/.openidc/oidc_wikitcms.json")
```

Comment:
You most probably need an `os.makedirs()` here?

Reply:
Also, I find it awkward to do this in `wiki-report.py` -- that's a file which you likely call as a developer locally, and then all of that is unexpected. Can you please move it to test/run?